packages/react-router-dom/modules/Match.js

```diff
@@ -0,0 +1,3 @@
+// Written in this round about way for babel-transform-imports
```

Comment:
[I changed the repo structure a bit](5bde02b80b68155ba84afeb4f424cb2e7bacde43). We won't need this file anymore since `react-router-dom`'s index.js does `export * from "react-router"`.